pkg/infra/teams/team3/loot_strat.go

```diff
@@ -75,9 +78,18 @@ func (a *AgentThree) HandleLootInformation(m message.TaggedInformMessage[message
 	}
 }
 
+// forcibly call at start of loot phase to begin proceedings
+func (a *AgentThree) RequestLootProposal(baseAgent agent.BaseAgent) { // put your logic here now, instead
+	sendProposal := rand.Intn(100)
+	if sendProposal > a.personality {
+		return
+	}
+	// general and send a loot proposal at the start of every turn
+	baseAgent.SendLootProposalToLeader(a.generateLootProposal())
+}
+
```

Comment:
This function now needs to be implemented. It is called at the start of each round and asks for the agent to send a loot proposal. The leader then takes this and distributes it for voting, which subsequently calls the `HandleLootProposal`